src/components/Map/USACountyMap.js

```diff
@@ -9,7 +9,7 @@ import { REVERSED_STATES } from 'common';
 import { useSummaries } from 'common/location_summaries';
 import { ScreenshotReady } from 'components/Screenshot';
 
-function getStateCode(stateName) {
+export function getStateCode(stateName) {
```

Comment:
nit: Would probably be better to move this into util/common/locations.ts if it's going to be a shared utility function.  But feel free to punt.